masq_lib/src/shared_schema.rs

```diff
@@ -50,6 +50,7 @@ pub const EARNING_WALLET_HELP: &str =
      (case-insensitive). If you already have a derivation-path earning wallet, don't supply this. \
      If you have supplied an earning wallet address before, either don't supply it again or be \
      careful to supply exactly the same one you supplied before.";
+pub const ENTRY_DNS_HELP: &str = "Specify whether to use entry DNS.";
```

Comment:
I'd like to see something here about `dns_utility` and DNS subversion, so that users can connect that practice to this parameter. Maybe something like this?

If you can't use HTTP proxying to reroute your client's traffic through MASQ's Node, you can specify `entry-dns` to start up MASQ's local DNS server; but you will also need to subvert your network stack's DNS traffic using `dns_utility`. This practice is not recommended but may be necessary in certain situations.